demo-projects/meetup/schema.js

```diff
@@ -20,7 +20,7 @@ const cloudinaryAdapter = new CloudinaryAdapter({
 });
 
 const access = {
-  userIsAdmin: ({ authentication: { item: user } }) => user && user.isAdmin,
+  userIsAdmin: ({ authentication: { item: user } }) => user && !!user.isAdmin,
```

Comment:
Was this intentional?

Reply:
Yep, it turns out if this function returns `undefined` then the access control doesn't behave correctly. This is probably a bug. `!!` forces the value to `false` which does behave correctly.